### PR TITLE
Ei 375 trigger emails for accounts without vrf

### PR DESCRIPTION
**Pull request by David-B-Read**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests.csproj [hunk 1]
```diff
@@ -4,6 +4,19 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="%2a%2a\**" />
+    <Content Remove="%2a%2a\**" />
```
Comment (ruslanss) [inline src/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests.csproj:9]: NOOOOOOOOOOOOO :((((((

File: src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/HandleBankDetailsRepeatReminderEmails.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        {
+            await RunBankDetailsRepeatReminderEmails(log);
+
+            return new OkResult();
+        }
+
+        private async Task RunBankDetailsRepeatReminderEmails(ILogger log)
+        {
+            log.LogInformation("Started bank details repeat reminder emails");
+            var cutOffDays = Convert.ToInt32(_configuration["BankDetailsReminderEmailsCutOffDays"]);
```
Comment (ruslanss) [inline src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/HandleBankDetailsRepeatReminderEmails.cs:41]: Convert.ToInt32(null) will return 0. so if the setting is null, we'll spam everyone. should we add some defence for the missing settings here?

File: src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/LegalEntities/IVendorRegistrationFormService.cs [hunk 3]
```diff
@@ -1,4 +1,6 @@
-﻿using System;
+﻿using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.LegalEntities.Types;
+using System;
+using System.Collections.Generic;
```
Comment (ruslanss) [inline src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/LegalEntities/IVendorRegistrationFormService.cs:3]: is this needed?

File: src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/LegalEntities/VendorRegistrationFormService.cs [hunk 4]
```diff
@@ -1,6 +1,8 @@
 ﻿using Newtonsoft.Json;
+using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.LegalEntities.Types;
 using SFA.DAS.EmployerIncentives.Infrastructure.Extensions;
 using System;
+using System.Collections.Generic;
```
Comment (ruslanss) [inline src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/LegalEntities/VendorRegistrationFormService.cs:5]: ?

**Review by ruslanss — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/LegalEntities/IVendorRegistrationFormService.cs (hunk 3 above)
Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/LegalEntities/IVendorRegistrationFormService.cs]: Hangover from when I added a new service method, which was then moved into the inner API domain handlers instead. Removed.

File: src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/HandleBankDetailsRepeatReminderEmails.cs (hunk 2 above)
Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/HandleBankDetailsRepeatReminderEmails.cs:42]: Added a default value

**Review by ruslanss — APPROVED** [review]

**Pull request merged**